test/830_cni_plugin_test.sh

```diff
@@ -48,12 +48,22 @@ assert_raises "exec_on $HOST1 c2 $PING $C1IP"
 # Now remove and start a new container to see if IP address re-use breaks things
 docker_on $HOST1 rm -f c2
 
-docker_on $HOST1 run --net=none --name=c3 -dt $SMALL_IMAGE /bin/sh
+C3=$(docker_on $HOST1 run --net=none --name=c3 -dt $SMALL_IMAGE /bin/sh)
```

Comment:
True, not useful here, as `weave ps c3` would have been equivalent.
However, it seems I need the full container ID for `curl` commands:

```
$ curl -X GET 127.0.0.1:6784/ip/c3
# No output

$ curl -X GET 127.0.0.1:6784/ip/8d5f808eee90
# No output

$ docker inspect --format="{{.Id}}" c3
8d5f808eee901c6fc5796aa894fb49727a295adf15eca35c4fd6721df19d1f3f

$ curl -X GET 127.0.0.1:6784/ip/8d5f808eee901c6fc5796aa894fb49727a295adf15eca35c4fd6721df19d1f3f
10.32.0.3/12
```